docs/RELEASE_NOTES.md

```diff
@@ -1,20 +1,28 @@
-# MongoDB Kubernetes Operator 0.7.8
+# MongoDB Kubernetes Operator 0.7.9
 
 ## Kubernetes Operator
 
 - Changes
-  - Fix a StatefulSet controller race condition with the mongodb-agent
-  - Fix merging label selectors when source is nil
-  - Improve logging verbosity
-  
+  - The Operator uses now the official [MongoDB Community Server images](https://hub.docker.com/r/mongodb/mongodb-community-server).
+    It is still possible to use the Docker Inc. images by altering the JSON configuration file:
+    ```asd
+          mongodb_image_name=mongo
+          mongodb_image_repo_url=docker.io
+    ```
+    Alternatively, it is possible to the Operator environmental variables to:
+    ```
+          MONGODB_IMAGE=mongo
+          MONGODB_REPO_URL=docker.io
+    ```
+    If the Operator is running using the default settings, the upgrade process will be automatic and seamless.
```

Comment:
How do I know if I'm not running the default settings? What settings?
And what do I do if I'm not?

Reply:
@dan-mckean Added more text. Please check it again when you find a free slot.

Reply:
Great! Thanks you @slaskawi - fine to resolve!